book.js

```diff
@@ -0,0 +1,43 @@
+var books = [
+    {
+      title: "The Design of Everyday Things",
+      author: "Don Norman",
+      alreadyRead: false
+    },
+    {
+      title: "The Most Human Human",
+      author: "Brian Christian",
+      alreadyRead: true
+    },
+    {
+      title: "In Search of Lost Time",
+      author: "Marcel Proust",
+      alreadyRead: true
+    },
+    {
+      title: "Ulysses",
+      author: "James Joyce",
+      alreadyRead: true
+    },
+    {
+      title: "The Great Gatsby",
+      author: "F. Scott Fitzgerald",
+      alreadyRead: true
+    }
+  ];
+  
+  
+const nwlistItem = document.querySelector('.listitem');
+nwlistItem.style.color = 'rebeccapurple';
+nwlistItem.fontFamily = 'Arial, sans-serif';

```

Comment:
This will change all the font in the whole page
```suggestion
document.querySelector('body').style.fontFamily = 'Arial, sans-serif';
```